### PR TITLE
Change recommended voltage regulator to pololu and fix typos on wiring page

### LINKED ISSUE #334: Add new voltage regulator
https://www.pololu.com/product/4082
-- moto moto, 2024

image

### PR DESCRIPTION
**Pull request by DeltaDizzy**
Fixes #334

### CODE REVIEW

File: source/docs/installation/wiring.rst [hunk 1]
```diff
@@ -18,19 +18,19 @@ On-Robot Wiring
...
 
-2. Add a breaker to relevant slot in your PDP/PDB
+2. Add a breaker to relevant slot in your PDP/PDH
 
 3. Run an ethernet cable from the passive POE injector to your network switch / radio (we *STRONGLY* recommend the usage of a network switch, see the networking section for more info.)
 
 Coprocessor without Passive POE
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-1a. Option 1: Get a micro USB (may be USB-C if using a newer Pi) to USB-A cable and plug the USB A side into a regulator like `this <https://www.amazon.com/KNACRO-Voltage-Regulator-Converter-Module/dp/B01HM12N2C/ref=sr_1_2>`_. Then, wire the regulator into your PDP/PDB and the Micro USB / USB C into your coprocessor.
+1a. Option 1: Get a micro USB (may be USB-C if using a newer Pi) to USB-A cable and plug the USB A side into a regulator like `this <https://www.pololu.com/product/4082>`_. Then, wire the regulator into your PDP/PDH and the Micro USB / USB C into your coprocessor.
```
Comment (mdurrani808) [inline source/docs/installation/wiring.rst:29]: The first part of this isn't true with this regulator. Include a wiring diagram.

File: source/docs/installation/wiring.rst [hunk 2]
```diff
@@ -18,19 +18,19 @@ On-Robot Wiring
...
 
 1b. Option 2: Use a USB power bank to power your coprocessor. Refer to this year's robot rulebook on legal implementations of this.
 
-1. Run an ethernet cable from your Pi to your network switch / radio (we *STRONGLY* recommend the usage of a network switch, see the networking section for more info.)
+2. Run an ethernet cable from your Pi to your network switch / radio (we *STRONGLY* recommend the usage of a network switch, see the networking section for more info.)
```
Comment (mdurrani808) [inline source/docs/installation/wiring.rst:33]: hyper link to networking

File: source/docs/installation/wiring.rst [hunk 3]
```diff
@@ -18,19 +18,19 @@ On-Robot Wiring
 Recommended: Coprocessor with Passive POE (Gloworm, Pi with SnakeEyes)
```
Comment (mdurrani808) [inline source/docs/installation/wiring.rst:18]: add LL

**Comment by mdurrani808** [comment]
add a note about the screw terminals it comes with those seem useful. sorry for lettign this sit for so long

**Review by mcm001 — APPROVED** [review]
Lgtm

**Pull request merged**